src/main/java/com/feywild/feywild/events/loot_modifiers/MineshaftChestLootModifier.java

```diff
@@ -0,0 +1,75 @@
+package com.feywild.feywild.events.loot_modifiers;
+
+import com.feywild.feywild.config.MiscConfig;
+import com.feywild.feywild.item.ModItems;
+import com.google.gson.JsonObject;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.util.GsonHelper;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.storage.loot.LootContext;
+import net.minecraft.world.level.storage.loot.predicates.LootItemCondition;
+import net.minecraftforge.common.loot.GlobalLootModifierSerializer;
+import net.minecraftforge.common.loot.LootModifier;
+import net.minecraftforge.registries.ForgeRegistries;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+public class MineshaftChestLootModifier extends LootModifier {
+
+    private final Item addition;
```

Comment:
What is this used for?

Reply:
```
 @Override
        public JsonObject write(MineshaftChestLootModifier instance) {

            JsonObject json = makeConditions(instance.conditions);
            json.addProperty("addition", ForgeRegistries.ITEMS.getKey(instance.addition).toString());
            return json;
        }
```

Reply:
Yes, the item is serialised. But it is never actually used for anything except being serialised. So why is it required?

Reply:
Without it it doesnt work...?

Reply:
It is not used currently. It is only serialised and then read again.

So I think I have to clarify the question: What SHOULD it be used for. So what purpose does that item field server?